jira/client.py

```diff
@@ -2672,7 +2662,7 @@ def create_temp_project_avatar(
         if size != size_from_file:
             size = size_from_file
 
-        params = {"filename": filename, "size": size}
+        params: Dict[str, Union[int, str]] = {"filename": filename, "size": size}
```

Comment:
its probably personal preference, but for me this doesnt improve readability. What would be the advantage of adding type annotation here?

Reply:
So now that we no longer use our own typehints for the requests methods (before resilient session had an override for the typehints), mypy wasn't happy as it inferred the type as `Dict[str, object]`